phonebuddies/EmailDraft.py

```diff
@@ -63,3 +63,14 @@ def draft_overdue_process_admin_reminder(email):
 """
 
         return EmailDraft(email, "ACTION NEEDED: AESOP Phone Buddy Reminder", message)
+
+    def draft_admins_without_buddy_emails(email):
+            message = f"""This is a friendly reminder that you are an administrator without a buddy. The dry-run of buddy emails has been sent, and 
+    any errors have been documented on the error tab here:
+    {DatabaseConnector.get_database_link}
```

Comment:
We'll need to finish up the code to document the errors (it kind of does it now, but it's not great), but this is a good way to do it in the future. LGTM.